sdk/agrifood/agrifood-farming-rest/test/public/sateliteFlow.spec.ts

```diff
@@ -1,66 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-import { FarmBeatsRestClient } from "../../src";
```

Comment:
Why are we removing these?

Reply:
These tests are already covered in farmHierarchy tests, so not needed now.